doc/man3/BN_rand.pod

```diff
@@ -37,7 +43,13 @@ If B<bits> is 1 then B<top> cannot also be B<BN_RAND_FLG_TOPTWO>.
 BN_rand_range() generates a cryptographically strong pseudo-random
 number B<rnd> in the range 0 E<lt>= B<rnd> E<lt> B<range>.
 
-The PRNG must be seeded prior to calling BN_rand() or BN_rand_range().
+BN_priv_rand() and BN_priv_rand_range() have, respectively, the same
+semantics as BN_rand() and BN_rand_range().  They are intended to be
+used for generating values that should remain private, and mirror the
+same difference between RAND_bytes() and RAND_priv_bytes().
+
+The PRNG must be seeded prior to calling BN_rand(), BN_rand_range(),
+BN_priv_rand(), or BN_priv_rand_range().
```

Comment:
@kroeckx Is the above paragraph still relevant? Or does it come from pre-1.1.0?

Reply:
@mspncp , can you remove the "ready" label, so that we can wait for comments from @kroeckx about this?

I have the feeling these 2 lines in 1.1.1 and 1.1.0 are not relevant anymore as the PRNG is automatically seeded for 3rd party applications upon initializing the library, am I wrong?

Reply:
@richsalz, please wait for the outstanding feedback from @kroeckx before merging (If you prefer, I can do the final merge, too. Just let me know.)

Reply:
As far as I know, since 1.1.0 we will try to initialize on first use. So this is not relevant for those versions. But only 1.1.1 has the priv functions.

Reply:
RAND_bytes has a comment about generating an error when it's not seeded. I wonder if we should add something like that here too.

Reply:
@kroeckx I should have fixed this and the suggestion about adding entries in the HISTORY sections in 7bd4945.
Can you please review them again?